examples/minimal-logs/main.tf

```diff
@@ -0,0 +1,156 @@
+locals {
+  az = data.aws_availability_zones.available.names[0]
+  tamr_vm_s3_actions = [
+    "s3:PutObject",
+    "s3:GetObject",
+    "s3:DeleteObject",
+    "s3:AbortMultipartUpload",
+    "s3:ListBucket",
+    "s3:ListObjects",
+    "s3:CreateJob",
+    "s3:HeadBucket"
+  ]
+}
+
+# Get available AZs
+data "aws_availability_zones" "available" {
+  state = "available"
+}
+
+# Get current Region
+data "aws_region" "current" {}
+
+# Get CloudWatchAgentServerPolicy
+data "aws_iam_policy" "cw-agent-server-policy" {
+  arn = "arn:aws:iam::aws:policy/CloudWatchAgentServerPolicy"
+}
+
+# Attach CloudWatchAgentServerPolicy to tamr-vm
+resource "aws_iam_role_policy_attachment" "cw-agent-server-policy" {
+  role       = module.tamr-vm.tamr_iam_role.tamr_instance_role_name
+  policy_arn = data.aws_iam_policy.cw-agent-server-policy.arn
+}
+
+# Set up VPC & subnet
+resource "aws_vpc" "tamr_vm_vpc" {
+  cidr_block           = "10.0.0.0/16"
+  enable_dns_support   = true
+  enable_dns_hostnames = true
+  tags                 = var.tags
+}
+
+resource "aws_subnet" "tamr_vm_subnet" {
+  vpc_id            = aws_vpc.tamr_vm_vpc.id
+  cidr_block        = "10.0.0.0/24"
+  availability_zone = local.az
+  tags              = var.tags
+}
+
+# Set up HBase logs bucket
+module "s3-bucket" {
+  source             = "git::git@github.com:Datatamer/terraform-aws-s3.git?ref=1.1.0"
+  bucket_name        = format("%s-tamr-module-test-bucket", var.name-prefix)
+  read_write_actions = local.tamr_vm_s3_actions
+  read_write_paths   = [""] # r/w policy permitting specified rw actions on entire bucket
+  additional_tags    = var.tags
+}
+
+# Create new EC2 key pair
+#resource "tls_private_key" "tamr_ec2_private_key" {
+#  algorithm = "RSA"
+#}
```

Comment:
Creating the key in the example seems ok.

One less hurdle to using the module.

Reply:
Sure!, will keep it. Thank you.